src/Platform/Microsoft.Testing.Platform/Builder/TestApplication.cs

```diff
@@ -179,7 +179,7 @@ private static async Task LogInformationAsync(
 #endif
         await logger.LogInformationAsync($"IsDynamicCodeSupported: {isDynamicCodeSupported}");
 
-        string moduleName = testApplicationModuleInfo.GetCurrentTestApplicationFullPath();
+        string moduleName = testApplicationModuleInfo.TryGetCurrentTestApplicationFullPath() ?? testApplicationModuleInfo.TryGetAssemblyName() ?? "unknown";
```

Comment:
This is used only for logging. So, we can tolerate null (e.g, in future when Wasm is supported). Then, we can fallback to just assembly name if we can find it, or otherwise just log it as "unknown"

Reply:
Maybe `<unknown-assembly>`

Reply:
Done